hoomd/md/pytest/test_methods.py

```diff
@@ -4,9 +4,9 @@
 from copy import deepcopy
```

Comment:
I would like to see a test added which confirms that this PR fixes the bug it is supposed to fix. There needs to be a test where we remove an integration method from a simulation, plug in a new one, and confirm that everything still works correctly.